assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -104,10 +113,10 @@ const OverflowButtons = styled.div`
   }
 `;
 
-const List = styled(Area).attrs({
+const PageList = styled(Reorderable).attrs({
```

Comment:
I'm still missing some space in the beginning of this container. I can't see the first separator before the first page - I can drop in the first position fine, but can't see the first separator, when the line carousel is scrollable.
![no-first-separator](https://user-images.githubusercontent.com/637548/78940140-5e52a380-7a83-11ea-8197-05b150883680.gif)

Reply:
Fixed